qiita_pet/templates/study_ajax/sample_summary.html

```diff
@@ -0,0 +1,78 @@
+{% from future.utils import viewitems %}
```

Comment:
Can we avoid as much as python code here? I don't think is strictly necessary to use viewitems here.